websites/K/Kahoot/dist/metadata.json

```diff
@@ -22,9 +26,10 @@
 	},
 	"url": [
 		"kahoot.it",
-		"play.kahoot.it"
+		"play.kahoot.it",
+		"create.kahoot.it"
 	],
-	"version": "2.0.10",
+	"version": "2.1.0",
```

Comment:
Would this PR be considered a minor or major update?

Reply:
Breaking